doc/reviewing_PR_guide.md

```diff
@@ -0,0 +1,91 @@
+
+The reason for this rule is similar to #1.  Very long PRs have a higher chance of introducing errors, and then those errors are hard to find.  Reviewers and bugfixers are human, and the longer the single block of changes, the harder it is to find the one that introduced the problem.
+
+Once again though, because it's caused problems before, some changes *can't* be broken into more than one PR, and that's okay if it's sufficiently justified.  In particular, we don't like dead code, so if a PR is long because it implements a new system and also adds a map or a monster that uses the new system, that's usually preferable to adding the map/monster in a follow-up.
+
+PRs that break this rule are likely to take longer to merge because they're going to be very hard to review, but we will eventually get to them.
+
+## 3. Outside sourced material
+
+CDDA is licensed under CC-BY-SA 4.0.  Unless you're really, really certain you understand *exactly* how this works, then you should not approve any PR using **any** material from outside the project: not even things like names, obvious references, et cetera.  Don't accept someone else's explanation for how it works, don't assume something is too trivial to count, and don't assume the material is fair use.  Just flag it as not meeting this criterion.
+
+Failing this guideline doesn't mean your PR is rejected, it just means we need to take a closer look.  The lead devs have been forced to become pretty expert at this and we'd like to ensure our rules are followed consistently, so if this is flagged, we'll come in and assess if it's okay.
+
+
+## 4. Changes consistent with the description
+
+While the PR description doesn't necessarily need to exhaustively outline every single line of code change, reading the PR description should clearly tell the reviewer what they're going to see in the changes.  There should not be any surprises in there.  If something was changed after the description was written, it needs to be updated.
+
+I find it helpful to think: if I was coming back to this PR in a year trying to remember what was in it to see if it might be responsible for a bug, could I use the PR description to figure that out?  If I *have* to read the code changes to even know if I'm in the right spot, there's a chance the description needs work.
+
+## 5. Maintainable and extendible
+
+This is obviously very much up to the reviewer.  "Janky" solutions to problems are solutions that try to work around a limitation of the codebase with funny tricks.  We see them often in JSON contributions, especially EOC things, but they are also very common in C++.  Sometimes they are allowable, but exploiting the code in strange ways has a high chance of breaking as soon as something else tries to rely on the code.  This rule becomes more important if the PR is changing something that other parts of the game are likely to rely on.
```

Comment:
```suggestion
This is obviously very much up to the reviewer. "Janky" solutions to problems are solutions that try to work around a limitation of the codebase with funny tricks.  We see them often in JSON contributions, especially EOC things, but they are also very common in C++. Sometimes they are allowable, but exploiting the code in strange ways has a high chance of breaking as soon as something else tries to rely on the code.  This rule becomes more important if the PR is changing something that other parts of the game are likely to rely on.
```

These aren't strings in the game, they do not need double spaces :p

Reply:
I keep it that way for consistency, it makes no difference in .md but some of our files throw an error.